.travis.yml

```diff
@@ -0,0 +1,12 @@
+---
+language: ruby
+before_script:
+  - "make"
+addons:
+  apt:
+    packages:
+      - 'aspell'
+      - 'aspell-en'
+script:
+  - "htmlproof out"
+  - '! cat *.md | aspell --dont-backup list --personal=./npm.en.pws | sort -u | sed "s/^/Not in Dictionaries: /" | grep "^"'
```

Comment:
@ashleygwilliams test scripts from Travis are here:

1. HTML checker (for hyperlinks, &c.)
2. Spell checker